ais-infra-alerting/README.md

```diff
@@ -0,0 +1,41 @@
+# Account Interventions Service - AIS Infra Alerting
+
+## Intro
+Consolidated alerting functionality to create a general infra alerting template with a subscription endpoint for PagerDuty and a single Chatbot configuration for all Slack alerts relating to AIS.
+
+## Prerequisites
+Export credentials for the AWS account where you want to deploy the application.
+You can use the following command to export credentials into your shell:
+```bash
+eval $(gds aws di-id-reuse-core-<environment>-admin -e)
```

Comment:
caveat wrt to envs and that prod and int will be different

Reply:
Thanks! Will add sso login instructions too 👍